### PR TITLE
Add Toggle component

### PR DESCRIPTION
**Pull request by felixjung**
Adds a `Toggle` component.

Component has optional props
- `label` (default `null`)
- `explanation` (default `null`)
- `margin` (default `true`)

Addresses #67.

### CODE REVIEW

**Comment by felixjung** [comment]
The explanation text is not styled properly, yet. Should be fixed when the resets from another branch are applied though (paragraph gets nasty user-agent style margins).

File: src/components/Setting/Setting.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        {label}
+      </SettingLabel>
+      {explanation && (
+        <SettingExplanation element="p" size="kilo">
+          {explanation}
+        </SettingExplanation>
+      )}
+    </SettingTextWrapper>
+  </SettingWrapper>
+);
```
Comment (mlent) [inline src/components/Setting/Setting.js]: So, something not clear to me is actually how we're going to show settings that are a choice between two different things.

For instance, payout settings.

This kind of setting only works when you turn something on/off but calling it a setting is a bit misleading when you think about settings we actually have.

@melmdesign

Comment (felixjung) [reply src/components/Setting/Setting.js]: I think those would be done with a `Selector` component, no? To me a setting is more like what this component represents. Hm But yeah, maybe `Setting` is too generic of a name.

**Comment by mlent** [comment]
Can we please just have one component called "Toggle" which optionally takes a label? If it gets a label it aligns nicely, if not, it's just a toggle?

Two components that basically look the same but one has a label and two different names makes no sense to me. Much less, "Setting" is too semantic for what this is - no doubt we will find more uses for it where Setting doesn't work and you're just misusing the thing.

**Comment by felixjung** [comment]
OK, please take a look. This should be done.

File: src/index.js [hunk 2]
```diff
@@ -2,6 +2,9 @@ import * as themes from './themes';
 import * as numbers from './util/numbers';
 import * as currency from './util/currency';
 
+// FIXME: these exports don't work unless we introduce a babel
+//        transform. This is future ES syntax.
+
```
Comment (connor-baer) [inline src/index.js]: Does this refer to `export { default as ... } from '...'`?

File: src/components/Toggle/components/Switch/Switch.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+const SwitchKnob = styled('div', { label: 'SwitchKnob' })(
+  knobBaseStyles,
+  knobOnStyles
+);
+
+/**
+ * A simple Switch component.
+ */
+const Switch = ({ on, onToggle }) => (
+  <SwitchTrack onClick={onToggle} on={on} role="switch" aria-checked={on}>
```
Comment (connor-baer) [inline src/components/Toggle/components/Switch/Switch.js]: Yay for accessibility! 💜

File: src/index.js (hunk 2 above)
Comment (felixjung) [reply src/index.js]: Yup.

**Pull request merged**